library/Zend/Db/Sql/Platform/Mysql/Ddl/AlterTableDecorator.php

```diff
@@ -135,7 +135,8 @@ protected function processAddColumns(PlatformInterface $adapterPlatform = null)
                 if ($insert) {
                     $j = isset($j) ? $j : 0;
                     $sql = substr_replace($sql, $insert, $insertStart[$j], 0);
-                    for (; $j < count($insertStart); ++$j) {
+                    $countInsertStart = count($insertStart);
```

Comment:
Count is a noun here not a verb, so it should be `$insertStartCount` instead of `$countInsertStart`.